alert-policies/flutter-android/AppCrashPercent.yml

```diff
@@ -0,0 +1,41 @@
+# Name of the alert
+name: App Crash Percent
```

Comment:
```suggestion
name: App Crash Rate
```

Reply:
I have updated it as per your comments.
Thank you...